### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by fstien**
Move installation and examples to bottom. Fix typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,5 +76,34 @@ install(OpenTracingServer) {
...
+### Gradle
+Add the following to your `dependencies` in your `build.gradle` 
+ 
+```
+implementation "com.zopa:ktor-opentracing:VERSION_NUMBER"
+```
+
+## Examples
+
+- For a simple example of ktor app instrumented with OpenTracing, see [ktor-opentracing-example](https://github.com/fstien/ktor-opentracing-example). This app uses span names passed explicitely to the `span` inline function.
```
Comment (hj1611) [inline README.md:101]: Typo - explicitly

File: README.md [hunk 2]
```diff
@@ -107,5 +76,34 @@ install(OpenTracingServer) {
...
+- For a simple example of ktor app instrumented with OpenTracing, see [ktor-opentracing-example](https://github.com/fstien/ktor-opentracing-example). This app uses span names passed explicitely to the `span` inline function. 
+
+- For automatic span naming using the class and method name, see [ktor-opentracing-span-naming-demo](https://github.com/fstien/ktor-opentracing-span-naming-demo).
+
+## Related Projects
+
+- For Ktor services using [kotlin-logging](https://github.com/MicroUtils/kotlin-logging), you can use [kotlin-logging-opentracing-decorator](https://github.com/fstien/kotlin-logging-opentracing-decorator) to enrich your spans with logs. 
+- If you are using [Exposed](https://github.com/JetBrains/Exposed), you can use [Exposed-OpenTracing](https://github.com/fstien/Exposed-OpenTracing) to instrument database transactions.
 
-#### [MIT](./LICENSE) License
```
Comment (hj1611) [inline README.md:111]: Do you not want to keep the licence information?

Comment (fstien) [reply README.md:111]: I think it looks a bit lame, we already have a LICENCE file 🙈

File: README.md (hunk 1 above)
Comment (fstien) [reply README.md]: yes pushed ! thanks

**Review by hj1611 — APPROVED** [review]

**Pull request merged**